src/cpp-utils/crypto/cryptopp_byte.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#ifndef _CRYPTOPP_BYTE_H
```

Comment:
Please use the pattern we have in other files for include guards, i.e. include the file path into the guard.